src/model/config.ts

```diff
@@ -0,0 +1,54 @@
+import BN from "bn.js";
+import { ConfigurableCorePool } from "../core/configurablePool";
+
+export class PoolConfig {
+    id: string;
+    tickSpacing: number;
+    token0: string;
+    token1: string;
+    fee: number;
+}
+
+export class Record {
```

Comment:
Is `Record` equivalent to `Event`?
If so, any reason that we are not just calling it `Event`?

Reply:
both OK. I kind of want to distinguish it from event emitted by a contract calling, since these events is also in the db now.

Reply:
I see, makes sense, `record` is good then.